docker-compose.yml

```diff
@@ -267,7 +280,39 @@ services:
           - prometheus.ly.fish.local
           - registry.ly.fish.local
           - s3.ly.fish.local
+          - hydra.ly.fish.local
     restart: unless-stopped
+  hydra:
+    build:
+      context: ./apps/hydra
+      dockerfile: Dockerfile
+    expose:
+      - 4444
+      - 4445
+    depends_on:
+      - hydra-migrate
+      - postgres
+      - mdns
+    environment:
+      - URLS_SELF_ISSUER=http://hydra.ly.fish.local
+      - URLS_CONSENT=http://jel.ly.fish.local/oauthprovider/consent
+      - URLS_LOGIN=http://jel.ly.fish.local/oauthprovider/login
+      - URLS_LOGOUT=http://jel.ly.fish.local/oauthprovider/logout
+      - DSN=postgres://docker:docker@postgres:5432/jellyfish?sslmode=disable&max_conns=20&max_idle_conns=4
+      - SECRETS_SYSTEM=youReallyNeedToChangeThis
```

Comment:
Not sure about this, see: https://www.ory.sh/docs/hydra/guides/secrets-key-rotation#rotation-of-hmac-token-signing-and-database-and-cookie-encryption-keys